fabric8-ui-dist/package.json

```diff
@@ -0,0 +1,215 @@
+{
+  "name": "fabric8-planner",
```

Comment:
Why is this file added here? It looks pretty much like something that exists in another place as well and maintaining multiple versions in different locations is too hard.

Reply:
accident!